Block/Js.php

```diff
@@ -585,6 +585,29 @@ public function getButtonCssStyles()
         return $style;
     }
 
+    /**
+     * Return CSS styles for bolt button
+     *
+     * @param string $buttonType
+     * 
+     * @return string
+     */
+    public function getVariantForInstantButton($buttonType = '')
```

Comment:
If buttonType == "" ; buttonVariant == "" and buttonPPCVarian == "PPC" this function returns
"&variant="